packages/mds-db/processors.ts

```diff
@@ -92,6 +93,25 @@ export async function insertMetrics(metric: MetricsTableRow) {
   return { ...metric, ...recorded_metric }
 }
 
+interface GetAllMetricsArgs {
+  start_time: Timestamp
+  end_time: number
+  provider_id: UUID | null
+  geography_id: UUID | null
+}
+
+export async function getAllMetrics({
```

Comment:
Kind of pedantic, but should this really be called getAllMetrics, seems to me like it's more of a getAllMetricsInWindow or getAllMetricsForProvider or something

Reply:
I think the name is fine